src/main/java/org/gridsuite/network/map/dto/mapper/BranchInfosMapper.java

```diff
@@ -256,7 +277,6 @@ private static List<TemporaryLimitData> toMapDataTemporaryLimit(@NonNull final C
                 .name(l.getName())
                 .acceptableDuration(l.getAcceptableDuration() == Integer.MAX_VALUE ? null : l.getAcceptableDuration())
                 .value(l.getValue() == Double.MAX_VALUE ? null : l.getValue())
-                .build())
-            .collect(Collectors.toList());
+                .build()).toList();
```

Comment:
be careful with that, sonar recommends it but it changes the behavior

Reply:
<img width="720" height="103" alt="Image" src="https://github.com/user-attachments/assets/362a5841-b808-4337-8500-e13f5af21095" />